cmd/kubeadm/app/phases/etcd/local.go

```diff
@@ -292,3 +265,44 @@ func getEtcdCommand(cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.A
 	command = append(command, kubeadmutil.BuildArgumentListFromMap(defaultArguments, cfg.Etcd.Local.ExtraArgs)...)
 	return command
 }
+
+func getAndWriteEtcdStaticPod(manifestDir string, patchesDir string, cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, nodeName string, initialCluster []etcdutil.Member, isDryRun bool) error {
```

Comment:
perhaps we should call it `prepareAndWriteEtcdStaticPod`?
since the "get" part is about getting the specs not the pod.

Reply:
changed the name to prepareAndWriteEtcdStaticPod, good suggestion.